commit 88236ac broke fsnapshot build

The recent PR #1599 breaks the compilation of `fsnapshot.cpp`:

```
fsnapshot.cpp: In function ‘void main_main()’:
fsnapshot.cpp:58:13: error: expected primary-expression before ‘||’ token
   58 |             ||igin = true;
      |             ^~
fsnapshot.cpp:58:15: error: ‘igin’ was not declared in this scope; did you mean ‘origin’?
   58 |             ||igin = true;
      |               ^~~~
      |               origin
make: *** [/home/zingale/development/amrex//Tools/GNUMake/Make.rules:67: fsnapshot.gnu.ex] Error 1```

--- a/Tools/Plotfile/fsnapshot.cpp
+++ b/Tools/Plotfile/fsnapshot.cpp
@@ -55,7 +55,7 @@ void main_main()
         } else if (name == "-l" || name == "--log") {
             do_log = true;
         } else if (name == "-g" || name == "--origin") {
-            ||igin = true;
+            origin = true;
         } else if (name == "-c" || name == "--coordinates") {
             location[0] = std::stod(amrex::get_command_argument(++farg));
             location[1] = std::stod(amrex::get_command_argument(++farg));

